src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/OpenDistroSecuritySSLPlugin.java

```diff
@@ -82,6 +82,7 @@
 //For ES5 this class has only effect when SSL only plugin is installed
 public class OpenDistroSecuritySSLPlugin extends Plugin implements ActionPlugin, NetworkPlugin {
 
+    public static final boolean OPENSSL_SUPPORTED = false;
```

Comment:
we should check this runtime. 
If java12 is available, this flag should be false otherwise it should be true. 

We should also check how we are building and packaging with? is it Java 8, 11 or 12?

Reply:
Made the changes. With github action, we are building the package with Java 11